runtime/doc/usr_05.txt

```diff
@@ -387,6 +387,27 @@ Only files of this filetype will use the settings from the plugin.  The <name>
 part of the plugin file doesn't matter, you can use it to have several plugins
 for the same filetype.  Note that it must end in ".vim".
 
+THE PLUG COMMAND                                                    *:plug*
+
+:plug[!] {spec}	        Add the plugin bundle defined by {spec} to 'runtimepath'.
+			{spec} can be a full path, the name of a directory in
+			the plugins folder (defined as `~/.vim/plugged/` by
+			default, see 'plugindir'), or it can have the form
+			`PREFIX/DIRNAME`.
+                        If {spec} is a full path, the command will work only
+                        if it exists in the filesystem.
+			If [!] is given, source the *.vim files in the plugin/
+			folder within {spec}.
+
+You can declare which paths to load plugins from using |:plug| from your
+|vimrc| file: >
+
+	plug ~/.config/nvim/plugged/nvim-plugin
```

Comment:
It looks like #3120 needs more work then even my last first iteration suggestion (`plugin#Initialize`+events, but not early plugins): it has undealt

1. Documentation. All references to `~/.vim` and `~/.nvim` should be found and replaced, option defaults changed.
2. Also XDG_\*_DIRS if this variant is accepted. This one may require tests, at least for colon iteration function.
3. Already failing tests.
4. Defaults for non-\*nix systems.